README.md

```diff
@@ -235,53 +243,124 @@ embeddings = co.embed_many(
 
-### 💫 Beyond Vector Search
-In order to perform well in production, modern GenAI applications require much more than vector search for retrieval. `redisvl` provides some common extensions that
-aim to improve applications working with LLMs:
 
-- **LLM Semantic Caching** is designed to increase application throughput and reduce the cost of using LLM models in production by leveraging previously generated knowledge.
+### Rerankers
+Integrate with popular reranking providers to improve the relevancy of the initial search results from Redis:
 
-    ```python
-    from redisvl.extensions.llmcache import SemanticCache
 
-    # init cache with TTL (expiration) policy and semantic distance threshhold
-    llmcache = SemanticCache(
-        name="llmcache",
-        ttl=360,
-        redis_url="redis://localhost:6379"
-    )
-    llmcache.set_threshold(0.2) # can be changed on-demand
 
-    # store user queries and LLM responses in the semantic cache
-    llmcache.store(
-        prompt="What is the capital city of France?",
-        response="Paris",
-        metadata={}
-    )
+## 💫 Extensions
+To improve production performance in GenAI applications, `redisvl` provides some common extensions:
 
-    # quickly check the cache with a slightly different prompt (before invoking an LLM)
-    response = llmcache.check(prompt="What is France's capital city?")
-    print(response[0]["response"])
-    ```
-    ```stdout
-    >>> "Paris"
-    ```
+### LLM Semantic Caching
+Increase application throughput and reduce the cost of using LLM models in production by leveraging previously generated knowledge with the [`SemanticCache`](https://www.redisvl.com/api/cache.html#semanticcache).
+
+```python
+from redisvl.extensions.llmcache import SemanticCache
+
+# init cache with TTL and semantic distance threshold
+llmcache = SemanticCache(
+    name="llmcache",
+    ttl=360,
+    redis_url="redis://localhost:6379",
+    distance_threshold=0.1
+)
+
+# store user queries and LLM responses in the semantic cache
+llmcache.store(
+    prompt="What is the capital city of France?",
+    response="Paris",
+    metadata={}
+)
+
+# quickly check the cache with a slightly different prompt (before invoking an LLM)
+response = llmcache.check(prompt="What is France's capital city?")
+print(response[0]["response"])
+```
+```stdout
+>>> "Paris"
+```
 
-    > Learn more about Semantic Caching [here](https://www.redisvl.com/user_guide/llmcache_03.html).
+> Learn more about [semantic caching]((https://www.redisvl.com/user_guide/llmcache_03.html)) for LLMs with `redisvl`.
 
-- **LLM Session Management (COMING SOON)** aims to improve personalization and accuracy of the LLM application by providing user chat session information and conversational memory.
-- **LLM Contextual Access Control (COMING SOON)** aims to improve security concerns by preventing malicious, irrelevant, or problematic user input from reaching LLMs and infrastructure.
+### LLM Session Management
 
```

Comment:
@justin-cechmanek can I get you're eye on these readme examples. Do you feel ok about these? Need to keep it simple, but also useful. Anything you would improve? Feel free to edit